src/index.html

```diff
@@ -2,17 +2,43 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Calendar</title>
-    <link
-      rel="stylesheet"
-      href="styles/index.scss"
-    />
+    <link rel="stylesheet" href="styles/main.scss" />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--start-day-sun">
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
+      <div class="calendar__item"></div>
```

Comment:
Consider using a loop in a template engine or JavaScript to generate the calendar items instead of hardcoding them. This will make the code more maintainable and scalable.